Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp

```diff
@@ -436,6 +452,25 @@ struct Scene_triangulation_3_item_priv {
   Scene_spheres_item *spheres;
   std::vector<Tr::Vertex> tr_vertices;
   Scene_intersection_item *intersection;
+  void set_intersection_enabled(bool b)
+  {
+    if (intersection)
+      intersection->setVisible(b);
+  }
+  bool is_intersection_enabled()
+  {
+    if (intersection)
+      return intersection->visible();
+    else
+      return false;
+  }
+  bool is_item_clip_box(int id)
+  {
+    if(dynamic_cast<Scene_edit_box_item*>(CGAL::Three::Three::scene()->item(id)))
```

Comment:
Because of that line, `scene_triangulation_3_item` must be linked with `scene_edit_box_item`.